src/Raven.Server/RavenServerStartup.cs

```diff
@@ -364,9 +365,8 @@ private static void MaybeSetExceptionStatusCode(HttpContext httpContext, ServerS
                 return;
             }
 
-            if (exception is LowMemoryException ||
+            if (exception.IsOutOfMemory() ||
```

Comment:
IsOutOfMemory does not return true for LowMemoryException? This is what you wanted?